CHANGELOG.md

```diff
@@ -7,22 +7,21 @@ This document contains a historical list of changes between releases. Only
 changes that impact end-user behavior are listed; changes to documentation or
 internal API changes are not present.
 
-Main (unreleased)
------------------
+## Main (unreleased)
```

Comment:
looks like the changelog was formatted. prettier on save? :D  we should make sure we only add the entry we need to this file.